File: POO/PY/PERSONA.py
class Persona:
    # Method -> Constructor
    def __init__(self, nombre, edad, carrera):
        self.nombre = nombre
        self.edad = edad
        self.carrera = carrera

    # Method -> Saludar 
    def saludar(self):
        print(f"Hola, mi nombre es {self.nombre} y tengo {self.edad} años. ")

    # Method -> Cumpleaños
    def cumpleaños(self):
        self.edad += 1
        print(f"¡Feliz cumpleaños! Ahora tengo {self.edad} años.")

    # Method -> Estudiar
    def estudiar(self):
        print(f"Estoy estudiando {self.carrera}.")

#Crear una instancia de la clase Persona
persona1 = Persona("Juan", 20, "Medicina")
persona2 = Persona("Maria", 21, "Odontologia")

#Llamar metodos
persona1.saludar()
persona2.saludar()
#persona1.estudiar()

File: POO/JS/PERSONA.js
class Persona {
    constructor(nombre, edad, carrera, universidad) {
      this.nombre = nombre;
      this.edad = edad;
      this.carrera = carrera;
      this.universidad = universidad;
    }
  
    saludar() {
      return 'Hola, mi nombre es ' + this.nombre + ' y tengo ' + this.edad + ' años.'
    }
  
    cumpleanios() {
      return '¡Felicidades! tienes ' + this.edad + ' años.';
    }
  

    estudiar() {
      return 'Estoy estudiando ' + this.carrera;

    }

    universidad1(){
        return 'Estoy estudiando en la' + this.universidad;
    }
    
  }

  // Crear una instancia de la clase Persona
  const persona = new Persona('Julio', 19, 'Ingeniería Industrial', ' Universidad Catolica Boliviana');
  
  // Inicializar los elementos HTML
  const saludar = document.getElementById('saludar');
  const edad = document.getElementById('edad');
  const carrera = document.getElementById('carrera');
  const universidad = document.getElementById('universidad');
  
  // Función para saludar
  function botonSaludar() {
    saludar.textContent = persona.saludar();
  }
  
  // Función para cumpleaños
  function botonCumpleanios() {
    edad.textContent = persona.cumpleanios();
  }
  
  // Función para estudiar
  function botonEstudiar() {
    carrera.textContent = persona.estudiar();
  }

  // Funcion para universidad
  function botonUniversidad1(){
    universidad.textContent = persona.universidad1();
  }